### PR TITLE
check to_v1 when version == 1

### PR DESCRIPTION
**Pull request by rink1969**

### CODE REVIEW

**Review by u2 — APPROVED** [review]

File: src/core/voteset.rs [hunk 1]
```diff
@@ -273,10 +273,27 @@ pub fn verify_tx_version(tx: &Transaction, version: u32) -> bool {
 
 // verify to, tx nonce and valid_until_block
 pub fn verify_tx(tx: &Transaction, height: u64) -> bool {
```
Comment (u2) [inline src/core/voteset.rs:275]: I guess it needs to check whether the block version is the same as tx.

Comment (rink1969) [reply src/core/voteset.rs:275]: These two vesion must be equal to current version on chain.
So we don't need to check they are same.

Comment (u2) [reply src/core/voteset.rs:275]: And the block version check?

Comment (rink1969) [reply src/core/voteset.rs:275]: yes
https://github.com/cryptape/cita-bft/blob/develop/src/core/cita_bft.rs#L930

**Pull request merged**